Makefile

```
@@ -505,6 +524,30 @@ $(BUILD_DIR)/assets/%.jpg.inc.c: assets/%.jpg
 $(BUILD_DIR)/%.schl.inc: %.schl
 	$(SCHC) $(SCHC_FLAGS) -o $@ $<
 
+# Audio
+
+AUDIO_BUILD_DEBUG ?= 0
+
+# first build samples...
+
+$(BUILD_DIR)/assets/audio/samples/%.half.aifc: assets/audio/samples/%.half.wav
+	$(SAMPLECONV) vadpcm-half $< $@
+
+$(BUILD_DIR)/assets/audio/samples/%.half.aifc: $(EXTRACTED_DIR)/assets/audio/samples/%.half.wav
+	$(SAMPLECONV) vadpcm-half $< $@
+ifeq ($(AUDIO_BUILD_DEBUG),1)
```

Most of the makefile does this kind of checks against 0 instead, so the consistency would be nice
```suggestion
ifneq ($(AUDIO_BUILD_DEBUG),0)
```

---

I think this is less readable and there's already `ifeq ($(ORIG_COMPILER),1)`, and in oot's makefile there's `ifeq ($(DEBUG),1)`

---

Fwiw, the main reason to use `ifneq` 0 is so that `0` means OFF while everything else means ON instead of `ifeq` 1 means `1` is ON and everything else is off.

---

Currently, out of the 10 boolean-like checks we have on this makefile, and only 1 is checked against `1` instead of against `0`.

In OoT there are 14 boolean checks, and 7 of them are checks against `1`.

So MM has a majority and OoT is half and half. I do think comparing against `0` is better since it is more boolean-like.

---

double negative in general is really hard to parse. Im not a fan of `ifndef` in general, even for things like NDEBUG.
I have no horse in this race but think that how tharo has it is much more readable